test/inductor/test_cutlass_backend.py

```diff
@@ -24,6 +24,12 @@
 from torch.testing._internal.inductor_utils import HAS_CPU, HAS_CUDA
 
 
+try:
+    from .test_aot_inductor import AOTIRunnerUtil
```

Comment:
is import path different based on the environment?

Reply:
yup I copied this from which supposedly makes it work with pytest

https://github.com/pytorch/pytorch/pull/130275/files

Reply:
it looks like `AOTIRunnerUtil` is defined in `test_aot_inductor_utils

Reply:
oh good catch!